server/test/unit/com/thoughtworks/go/remote/work/BuildWorkTest.java

```diff
@@ -629,4 +613,30 @@ private void createBuildWorkWithJobPlan(JobPlan jobPlan) throws Exception {
         );
         buildWork = new BuildWork(buildAssignment);
     }
+
+    @Test
+    public void shouldReportEnvironmentVariables() throws Exception {
+        buildWork = (BuildWork) getWork(WITH_ENV_VAR, PIPELINE_NAME);
+
+        buildWork.doWork(agentIdentifier, buildRepository, artifactManipulator, environmentVariableContext, AgentRuntimeInfo.fromAgent(agentIdentifier, "cookie", null), packageAsRepositoryExtension, scmExtension, taskExtension);
+
+        String consoleOut = artifactManipulator.consoleOut();
+
+        assertThat(trimTimeStamp(consoleOut), containsString("[go] setting environment variable 'GO_SERVER_URL' to value 'somewhere-does-not-matter'\n"
+                + "[go] setting environment variable 'GO_TRIGGER_USER' to value ''\n"
+                + "[go] setting environment variable 'GO_PIPELINE_NAME' to value 'pipeline1'\n"
```

Comment:
I guess windows user will have some difficulties. They will say I don't see anything "export" related in this dialog -- 
![image](https://cloud.githubusercontent.com/assets/10283/12497157/e2d2de02-c04e-11e5-8da5-254c06307378.png)